sysutils/asdf/Portfile

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        asdf-vm asdf 0.10.2 v
+categories          sysutils devel
+platforms           darwin
```

Comment:
`platforms darwin` is implicit in MacPorts now, so you can remove this line.

Reply:
Got it, removed.